consensus/polybft/contracts_initializer.go

```diff
@@ -404,3 +430,8 @@ func callContract(from, to types.Address, input []byte, contractName string, tra
 func isNativeRewardToken(cfg PolyBFTConfig) bool {
 	return cfg.RewardConfig.TokenAddress == contracts.NativeERC20TokenContract
 }
+
+// isNativeStakeToken return true in case a native token is used for staking
+func IsNativeStakeToken(stakeTokenAddr types.Address) bool {
```

Comment:
No need to export it:

```suggestion
func isNativeStakeToken(stakeTokenAddr types.Address) bool {
```